### ISSUE TITLE
Mac M1

### ISSUE THREAD

**Issue by drveni** [supermerill/SuperSlicer#2675]
### What happened?

wont start

### Project file & How to reproduce

SuperSlicer_2.3.57.12_macos_220401 open correct and start

SuperSlicer_2.4.58.2_macos_220401 wont start...

### Version

2.4.58.2

### Operating system

M1 mac

### Printer model

non

**Comment by foxmckraut** [comment]
try https://github.com/supermerill/SuperSlicer/issues/2543

**Comment by supermerill** [comment]
can you test this build for macos arm? https://mega.nz/file/9gwyHYKS#-BJPvMGzupZPLbnMqWBF3elMV7BLzXXqnBZ6IXNO2HU
I tried with zstd uninstalled, so it shouldn't be needed, but I'm not 100% confident.
i guess it still need the xattr thingy to be launch on another computer than the one that built it?

**Comment by drveni** [comment]
> can you test this build for macos arm? https://mega.nz/file/9gwyHYKS#-BJPvMGzupZPLbnMqWBF3elMV7BLzXXqnBZ6IXNO2HU I tried with zstd uninstalled, so it shouldn't be needed, but I'm not 100% confident. i guess it still need the xattr thingy to be launch on another computer than the one that built it?


it works. I started. I'll try to test later slice and menu. i don't have zstd installed. xattr thingy im not shure what it is I just ran over the old last one that worked.

**Comment by Juggeli** [comment]
> can you test this build for macos arm? https://mega.nz/file/9gwyHYKS#-BJPvMGzupZPLbnMqWBF3elMV7BLzXXqnBZ6IXNO2HU
> I tried with zstd uninstalled, so it shouldn't be needed, but I'm not 100% confident.
> i guess it still need the xattr thingy to be launch on another computer than the one that built it?

This has been running great. I have sliced about dozen models for several printers without any hiccups. It needed the xattr command to run tho.

**Comment by supermerill** [comment]
latest nightly:
https://mega.nz/file/NxIigRwR#hKgPdD1LykfLO3cwrUt_Qzs4BvcDVgh94_KDSNcV788

**Comment by drveni** [comment]
> latest nightly: https://mega.nz/file/NxIigRwR#hKgPdD1LykfLO3cwrUt_Qzs4BvcDVgh94_KDSNcV788

i gotta problem with this after dowlonad and install. startup app says Superslices is damaged and cant be opened... you should move it to the bin

**Comment by foxmckraut** [comment]
> > latest nightly: https://mega.nz/file/NxIigRwR#hKgPdD1LykfLO3cwrUt_Qzs4BvcDVgh94_KDSNcV788
> 
> i gotta problem with this after dowlonad and install. startup app says Superslices is damaged and cant be opened... you should move it to the bin

do the xatr thing...

try:
sudo xattr -cr "/Volumes/MacOS SSD/Applications/SuperSlicer.app"
or

sudo xattr -cr "/Volumes/Macintosh HD/Applications/SuperSlicer.app"

in terminal

**Comment by drveni** [comment]
> sudo xattr -cr "

sorry nothing is happened....

first download liked gread..  dragged the file where i want to install clicked and launched..

**Comment by foxmckraut** [comment]
> > sudo xattr -cr "
> 
> sorry nothing is happened....
> 
> first download liked great.. dragged the file where i want to install clicked and launched..

if you drag it anywhere else than to aplication you need to correct the path i posted above.

**Comment by drveni** [comment]
> > > sudo xattr -cr "
> > 
> > 
> > sorry nothing is happened....
> > first download liked great.. dragged the file where i want to install clicked and launched..
> 
> if you drag it anywhere else than to aplication you need to correct the path i posted above.

yes i did that....yes I know that and I understand ... looks some logs from mac .. and I don't see anything special.

**Comment by foxmckraut** [comment]
download > drag to aplication > sudo xattr -cr "/Volumes/Macintosh HD/Applications/SuperSlicer.app" > launch SS

**Comment by drveni** [comment]
> download > drag to aplication > sudo xattr -cr "/Volumes/Macintosh HD/Applications/SuperSlicer.app" > launch SS
> 
> correct?

yes!!


from terminal 

tihomir@Tihomirs-Mac-mini SuperSlicer.app % pwd
/Users/tihomir/Desktop/SuperSlicer.app
tihomir@Tihomirs-Mac-mini SuperSlicer.app % sudo xattr -cr "/Users/tihomir/Desktop/SuperSlicer.app"          
Password:
tihomir@Tihomirs-Mac-mini SuperSlicer.app %

**Comment by foxmckraut** [comment]
seems correct but i cant say if it will work on desktop...you tried also in aplications folder?

thats i only thing i could say is different between your mac an mine...

**Comment by drveni** [comment]
> seems correct but i cant say if it will work on desktop...you tried also in aplications folder?
> 
> thats i only thing i could say is different between your mac an mine...
<img width="337" alt="Screenshot 2022-05-02 at 20 38 51" src="https://user-images.githubusercontent.com/62386210/166305734-b97a7e21-5562-4dcc-9ba2-37b38e643bb9.png">

installed to aplication folder... and same error...

**Comment by foxmckraut** [comment]
installed in aplication followed by sudo xattr -cr "/Volumes/Macintosh HD/Applications/SuperSlicer.app"  in terminal correct?

you can also try pressing option button+leftclick and than open.

**Comment by drveni** [comment]
> 

my option button + right click nothinh opened... when i enter in superslicer.app i see all directoy and subdirectoys... and files.. all seems ok 

are you corrected upload files? try download 3 times same file.. same error.. im not shure.. but first lik what workend instaled od another mount drive and works... this gotta error. i really dont know. what log to look?? is there another log file somewhere? I don't see anything from the mac?.

**Comment by foxmckraut** [comment]
> > 
> 
> my option button + right click nothinh opened... when i enter in superslicer.app i see all directoy and subdirectoys... and files.. all seems ok
> 
> are you corrected upload files? try download 3 times same file.. same error.. im not shure.. but first lik what workend instaled od another mount drive and works... this gotta error. i really dont know. what log to look?? is there another log file somewhere? I don't see anything from the mac?.

i have an macbook air 2020 m1 and downloaded it from the link above...done the things i told you and it worked properly...

**Comment by drveni** [comment]
> > > 
> > 
> > 
> > my option button + right click nothinh opened... when i enter in superslicer.app i see all directoy and subdirectoys... and files.. all seems ok
> > are you corrected upload files? try download 3 times same file.. same error.. im not shure.. but first lik what workend instaled od another mount drive and works... this gotta error. i really dont know. what log to look?? is there another log file somewhere? I don't see anything from the mac?.
> 
> i have an macbook air 2020 m1 Macosx 12.3.1 and downloaded it from the link above...done the things i told you and it worked properly.. <img alt="Bildschirmfoto 2022-05-02 um 20 59 15" width="501" src="https://user-images.githubusercontent.com/101701200/166308374-8d18902f-b751-4982-b26f-a87017b71d46.png"> .

im on Mac mini (M1, 2020) 12.3.1.. i realy dont know why
